qutebrowser/misc/utilcmds.py

```diff
@@ -86,6 +86,19 @@ def repeat(times: int, command, win_id):
         commandrunner.run_safely(command)
 
 
+@cmdutils.register(maxsplit=1, hide=True, no_cmd_split=True,
+                   no_replace_variables=True)
+@cmdutils.argument('win_id', win_id=True)
+def run_with_count(times: int, command, win_id):
```

Comment:
Maybe `:run-with-count` should take a count too and multiply the two? That way you could e.g. bind `j` to `:run-with-count 2 down` to get a bigger scroll step, but then still use `10j`.